ivanti_service_manager/plugin.spec.yaml

```diff
@@ -734,6 +734,707 @@ types:
+      type: string
+      required: false
+    FulfillmentPlanLink_RecID:
+      title: Fulfillment Plan Link Record ID
+      description: Fulfillment plan link record ID
+      type: string
+      required: false
+    FulfillmentPlanLink:
+      title: Fulfillment Plan Link 
+      description: Fulfillment plan link
+      type: string
+      required: false
+    Cost_Currency:
+      title: Cost Currency
+      description: Cost currency
+      type: string
+      required: false
+    Cost_CurrencyValid:
+      title: Cost Currency Valid
+      description: Cost currency valid
+      type: string
+      required: false
+    Cost:
+      title: Cost
+      description: Cost
+      type: string
+      required: false
+    CostPerMinute_Currency:
+      title: Cost Per Minute Currency
+      description: Cost per minute currency
+      type: string
+      required: false
+    CostPerMinute_CurrencyValid:
+      title: Cost Per Minute Currency Valid
+      description: Cost per minute currency valid
+      type: string
+      required: false
+    CostPerMinute:
+      title: Cost Per Minute
+      description: Cost per minute
+      type: string
+      required: false
+    DefaultApprover_Valid:
+      title: Default Approver Valid
+      description: Default approver valid
+      type: string
+      required: false
+    DefaultApprover:
+      title: Default Approver
+      description: Default approver
+      type: string
+      required: false
+    SLA:
+      title: SLA
+      description: SLA
+      type: string
+      required: false
+    SLADisplayText:
+      title: SLA Display Text
+      description: SLA display text
+      type: string
+      required: false
+    OrganizationUnitID:
+      title: Organization Unit ID
+      description: Organization unit ID
+      type: string
+      required: false
+    ReadOnly:
+      title: Read Only
+      description: Is read only
+      type: boolean
+      required: false
+    ProfileLoginId:
+      title: Profile Login ID
+      description: Profile login ID
+      type: string
+      required: false
+    TotalTimeSpent:
+      title: Total Time Spent
+      description: Total time spent
+      type: integer
+      required: false
+    IsDSMTaskExisted:
+      title: Is DSM Task Existed By
+      description: Is DSM task existed
+      type: boolean
+      required: false
+    DraftPrice_Currency:
+      title: Draft Price Currency
+      description: Draft price currency
+      type: string
+      required: false
+    DraftPrice_CurrencyValid:
+      title: Draft Price Currency Valid
+      description: Draft price currency valid
+      type: string
+      required: false
+    DraftPrice:
+      title: Draft Price
+      description: Draft price
+      type: string
+      required: false
+    IsDraft:
+      title: Is Draft
+      description: Is draft
+      type: boolean
+      required: false
+    RequestOfferingWFTrigger:
+      title: Request Offering WF Trigger
```

Comment:
Spell out WF in the title.

Reply:
I've updated the abbreviations in the titles, and ran `make regenerate` to update the schemas. Also pulled in latest master changes from master.